app/utils/computed-helpers.js

```diff
@@ -41,8 +41,8 @@ export const computedSegmentedLink = function(property) {
     },
     set(key, value) {
       const finalLink = values(value).join('');
-
-      if (finalLink && isValidUrl(finalLink.trim())) {
+      const isUserNamePresent = finalLink.split('.com/')[1];
+      if (finalLink && isValidUrl(finalLink.trim()) && isUserNamePresent) {
```

Comment:
this will ensure we are saving only if we have something after `htps://facebook.com/`